spec/update-package-dependencies-spec.coffee

```diff
@@ -23,7 +23,8 @@ describe "Update Package Dependencies", ->
 
       expect(mainModule.runBufferedProcess).toHaveBeenCalled()
       [{command, args, options}] = mainModule.runBufferedProcess.argsForCall[0]
-      expect(command).toMatch(/apm$/)
+      expect(command).toMatch(/\/apm$/) unless process.platform is 'win32' 
+      expect(command).toMatch(/\apm.cmd$/) if process.platform is 'win32'
```

Comment:
Thanks, I did have that and at some point when I switched from a single expect with a ternary platform regex to the multi-line one I lost it :(